source/views/sis/course-search/lib/format-day.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+
+export function formatDay(day: string) {
+	switch (day) {
+		case 'Mo':
+			return 'Monday'
+		case 'Tu':
+			return 'Tuesday'
+		case 'We':
+			return 'Wednesday'
+		case 'Th':
+			return 'Thursday'
+		case 'Fr':
+			return 'Friday'
+		default:
+			return 'unknown'
```

Comment:
`return 'Unknown'` (capitalized)

Reply:
For the sake of consistency, you [asked for “Unlnown Term”](https://github.com/StoDevX/AAO-React-Native/pull/2293#discussion_r167254894) so this should be “Unlnown Day”

Reply:
Wait. We should include `Sa` and `Su`, too.